internal/selfmonitor/config/fluent_bit_rule_builder.go

```diff
@@ -74,6 +71,24 @@ func (rb fluentBitRuleBuilder) bufferFullRule() Rule {
 	}
 }
 
+func (rb fluentBitRuleBuilder) noLogsDeliveredRule() Rule {
+	receiverReadExpr := rate(metricFluentBitInputBytesTotal, selectService(fluentBitMetricsServiceName)).
+		sumBy(labelPipelineName).
+		greaterThan(0).
+		build()
+
+	exporterNotSentExpr := rate(metricFluentBitOutputProcBytesTotal, selectService(fluentBitMetricsServiceName)).
+		sumBy(labelPipelineName).
+		equal(0).
```

Comment:
Maybe it's a stupid idea, but wouldn't it be safer to use "<="

Reply:
Rate cannot be negative